keps/sig-node/4639-oci-volume-source/README.md

```diff
@@ -586,8 +589,13 @@ feature cannot be used. Pods using the new `VolumeSource` combined with a not
 supported container runtime version will fail to run on the node, because the
 `Mount.host_path` field is not set for those mounts.
 
-For security reasons, volume mounts should set the [`noexec`] and `ro`
-(read-only) options by default.
+For the most conservative security approach, image volumes should be mounted by
+runtimes with enabled `rro` (recursive read-only) and `noexec` flag. Runtimes
+may support writable volumes in the same way as the running container image
+contents are writable (for example by utilizing overlayfs). This enables users
+to modify the image volume contents in a non-persistent and container isolated
```

Comment:
Is it isolated to the _container_ or isolated to the _pod_?

Reply:
Yes my initial proposal was intended for container isolation. I think pod isolation would make even more sense from a use case perspective. Changed that, thank you!